src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -27,56 +27,78 @@
       :on-press            #(rf/dispatch [:navigate-to :wallet-address-watch])
       :add-divider?        true}]]])
 
-(def account-cards
-  [{:name                "Account 1"
-    :balance             "€0.00"
-    :percentage-value    "€0.00"
-    :customization-color :blue
-    :type                :empty
-    :emoji               "🍑"
-    :on-press            #(rf/dispatch [:navigate-to :wallet-accounts])}
-   {:customization-color :blue
-    :on-press            #(rf/dispatch
-                           [:show-bottom-sheet {:content new-account}])
-    :type                :add-account}])
+(def add-account-placeholder
+  {:customization-color :blue
+   :on-press            #(rf/dispatch [:show-bottom-sheet {:content new-account}])
+   :type                :add-account})
 
 (def tabs-data
   [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
 
+(defn get-balance
+  [balances address]
+  (.toFixed (->> balances
+                 (filter #(= (:address %) address))
+                 first
+                 :balance)
+            2))
+
+(defn account-cards
+  [accounts loading? balances]
+  (let [refactored-accounts (mapv (fn [account]
+                                    (assoc account
+                                           :type                :empty
+                                           :customization-color :blue
```

Comment:
should `type` be `:emtpy?` 

customization color should not be hardcoded - can you check if `account` has a color and if so let's use that here 👍

Reply:
feel free to do this in a follow up too 👍 
will be great to have this pr in as some other issues will benefit from this code added :)

Reply:
The type is actually `empty`. Base on the designs for this component `src/quo2/components/wallet/account_card/view.cljs`.

Reply:
The color is also done